DotNetAppSqlDb/Controllers/TodosController.cs

```diff
@@ -86,7 +85,7 @@ public ActionResult Edit(Guid? id)
         [ValidateAntiForgeryToken]
         public ActionResult Edit([Bind(Include = "id,Description,CreatedDate")] Todo todo)
```

Comment:
It's needed
Store update, insert, or delete statement affected an unexpected number of rows (0). Entities may have been modified or deleted since entities were loaded. See http://go.microsoft.com/fwlink/?LinkId=472540 for information on understanding and handling optimistic concurrency exceptions.